libraries/AP_HAL_ChibiOS/hwdef/BOTWINGF405/Readme.md

```diff
@@ -0,0 +1,165 @@
+
+# BOTWINGF405 Flight Controller
+
+The BOTWINGF405 is a compact, high-performance flight controller developed for fixed-wing and FPV applications. Designed for reliability and flexibility, it integrates essential sensors and features for smooth flight and rich telemetry, produced by [BOTLAB DYNAMICS](https://www.botlabdynamics.store/).
+
+
+![BOTWINGF405 Board](Layout_top_1.png "BOTWINGF405-TOP-1")
+
+![BOTWINGF405 Board](Layout_top_2.png "BOTWINGF405-TOP-2")
+
+![BOTWINGF405 Board](Layout_bottom_1.png "BOTWINGF405-BOTTOM-1")
+
+![BOTWINGF405 Board](Layout_bottom_2.png "BOTWINGF405-BOTTOM-2")
+
+## Features
+
+### Processor
+
+* STM32F405RGT6, 168 MHz, 1MB flash
+* 24MHz external crystal
+
+
+### Sensors
+
+* ICM42688P Accelerometer/Gyroscope
+* DPS310 Barometer
+* Optional External Compass (AK8963 supported)
+
+
+### Power
+
+* 2S–6S LiPo input with onboard voltage and current monitoring
+* BEC Outputs:
+  * 5V @ 2.0A
+  * 9V @ 2.0A, GPIO controlled 
+
+
+### Storage
+
+* 32MB onboard dataflash for logging
+* JEDEC-compatible SPI Flash (W25Q256)
+
+
+### Interfaces
+
+* 6x UARTs (for GPS, telemetry, RC, camera, etc.)
+* I2C port for barometer and external compass
+* USB OTG port (USB connector)
+* 5 configured PWM outputs (4 for motors, 1 for RGB LED)
+* 1x RC input (SBUS, PPM or CRSF/ELRS selectable)
+
+
+### External Connections
+
+* 6-pin JST-GH for GPS/Compass
+* 6-pin JST-GH for HD VTXX or other peripherals
+* 8-pin JST-GH for ESC
+* 4-pin JST-GH for Receiver input
+* 5-pin JST-GH for User
+* 5-pin JST-GH for Camera Input
+* 4 x 4-pin JST-GH for LED Strip-NEOPIXEL
+
+
+## Wiring Diagram
+
+![BOTWINGF405 Wiring](wiring_diagram.png)
+
+
+## UART Mapping
+
+The UARTs default protocol and serial port assigments are:
+
+* SERIAL0 -> USB
+* SERIAL1 -> USART1 (DMA capable,RX tied to SBUS RC input and pin used as an interrupt input, but can be used as normal UART if :ref:`BRD_ALT_CONFIG =1` ) 
+* SERIAL2 -> USART2 (ESC Telemetry, TX DMA Capable)
```

Comment:
ESC is RX only so DMA on TX seems moot. In general I would only reference DMA on UARTS that have bi-directional DMA